packages/editor/src/components/post-publish-panel/style.scss

```diff
@@ -6,7 +6,7 @@
 	// Ensure the post-publish panel accounts for the header and footer height.
 	min-height: calc(100% - #{ $header-height + 84px });
 
-	.components-spinner {
+	> .components-spinner {
```

Comment:
This seems quite specific here and this component is public API. Should we instead style your new Spinner instead? 🤔

Reply:
So this is styling another spinner in the same pannel and we want the new spinner to have no extra styling. Changed it here so this extra CSS only affects the existing spinner.